lukefi/metsi/app/simulator.py

```diff
@@ -26,14 +26,21 @@ def run_stands(stands: StandList,
 
     retval: dict[str, list[ForestOpPayload]] = {}
     for stand in stands:
-        overlaid_stand = LayeredObject[ForestStand](stand)
-        overlaid_stand.reference_trees = [LayeredObject[ReferenceTree](tree) for tree in overlaid_stand.reference_trees]
-        overlaid_stand.tree_strata = [LayeredObject[TreeStratum](stratum) for stratum in overlaid_stand.tree_strata]
+        overlaid_stand: PossiblyLayered[ForestStand]
```

Comment:
Wondering if what happens here is clear enough. Refering to docstring it does not bring out any noticable information about the overlay. The LayeredObject neither has a clear docstring to clearify its usage.

I would suggest we add some clearification in form of a docstring or separate comment.